pkgs/tools/misc/bartib/default.nix

```diff
@@ -2,16 +2,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "bartib";
-  version = "1.0.1";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "nikolassv";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0ph3rsrhcyi272bv5018pw185zn7fvp5fqj24yh9rjrz8x7iawib";
+    sha256 = "sha256-eVLacxKD8seD8mxVN1D3HhKZkIDXsEsSisZnFbmhpSk=";
   };
 
-  cargoSha256 = "sha256-1ZFwX7NKIainer7o9dIMxwyycdGW8K9euLHad/tF95w=";
+  cargoSha256 = "sha256-MVQk055u1Ezs3h7rBe6qSia+K+5+TPiYPHdzZ+53fdM=";
+  cargoPatches = [
+    ./update_lock-version.patch
+  ];
```

Comment:
I think you can use the following instead:
```nix
  # Cargo.lock is outdated
  preConfigure = ''
    cargo metadata --offline
  '';
```

Reply:
huh that is cool, did not know that
thanks!